.travis.yml

```diff
@@ -162,6 +162,7 @@ deploy:
     - kubeless-${TRAVIS_TAG}.yaml
     - kubeless-rbac-${TRAVIS_TAG}.yaml
     - kubeless-openshift-${TRAVIS_TAG}.yaml
+    - kafkazk-${TRAVIS_TAG}.yaml
```

Comment:
note that the file will be called `kafkazk.yaml`, you need to rename it (with the other yamls, line 153)

Reply:
you still need to generate this file, isn't it?

Reply:
Yes, it's generated via `make all-yaml` at this line https://github.com/kubeless/kubeless/pull/570/files#diff-b67911656ef5d18c4ae36cb6741b7965R40

Reply:
I would rename this file `kafka-zookeeper.yaml` and the associated `kafka-zookeeper.jsonnet`

they seem to be generated properly in the makefile fwiw.